lib/knockout.dragdrop.js

```diff
@@ -6,6 +6,9 @@
     if (typeof define === "function" && define.amd) {
         // AMD anonymous module with hard-coded dependency on "knockout"
         define(["knockout"], factory);
+    } else if(typeof require === "function" && typeof exports === "object" && typeof module === "object") {
+        // CommonJS or Node: hard-coded dependency on "knockout"
+        factory(require("knockout"), exports);
```

Comment:
The standard UMD wrapper with support for node.js looks like this: https://github.com/umdjs/umd/blob/master/templates/returnExports.js#L21-L25 -- could we use that instead? (ie. look for `module.exports` instead).

Also, `factory` doesn't use the 2nd argument, so there's no need to pass that.

Thirdly, please include a space between `if` and `(` :)